bodhi/server/util.py

```diff
@@ -516,7 +516,18 @@ def status2html(context, status):
         'obsolete': 'default',
         'processing': 'info',
     }[status]
-    return "<span class='label label-%s'>%s</span>" % (cls, status)
+    status_desc = {
+        'pending': 'The update has not yet been pushed to the testing or stable repositories.',
+        'testing': 'The package is in the updates-testing repository for people to test.',
```

Comment:
I would say testing instead of updates-testing, because updates-testing is specific to Fedora's dnf configs (there are other Bodhi deployments that might call their testing repos something else.)